package.json

```diff
@@ -30,5 +33,8 @@
 	},
 	"peerDependencies": {
 		"react-dom": "^18.2.0"
+	},
+	"publishConfig": {
+		"provenance": true
```

Comment:
This will force your npm package to publish using GitHub. This gives your users confidence that what is being published is the same as what is on GitHub.
<img width="251" alt="image" src="https://github.com/Decidable-AS/fallback-email/assets/25374940/435c2d90-c7af-4500-9000-b35fc8c8293a">